### PR TITLE
add_Gaus_task

### PR DESCRIPTION
**Pull request by ChaikaS**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,124 +1,72 @@
...
+
+
+// for (let i = 0; i < arr.length; i++) {
+//    let value = arr[i];
+//    if (result[value] != undefined)
+//       result[value]++;
+//    else
+//       result[value] = 1;
+// }
+
```
Comment (mikalaider) [inline script.js]: Please remove unused code

File: script.js [hunk 2]
```diff
@@ -1,124 +1,72 @@
...
+}
+
+console.log(arr);
+document.write(arr + ' наш сгенерированный массив <br>')
+document.write('<br>')
+
+
+
+arr.forEach((item) => {
+   if (result[item] != undefined)
```
Comment (mikalaider) [inline script.js]: No need to check for undefined

File: script.js [hunk 3]
```diff
@@ -1,124 +1,72 @@
...
+   return Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2 | 0;
+};
+
+
+let n = 10;
+let arr = []
+let result = {}
+for (let i = 0; i < n; i++) {
+   arr.push(randn_bm())
+}
```
Comment (mikalaider) [inline script.js]: Please put all this logic in a separate function

File: script.js [hunk 4]
```diff
@@ -1,124 +1,72 @@
...
+//       result[value]++;
+//    else
+//       result[value] = 1;
+// }
+
+
+
+for (let key in result)
+   document.write('число ' + key + ' == ' + result[key] + ' раз(а) <br>');
+
```
Comment (mikalaider) [inline script.js]: Please put all this logic in a separate function

Comment (ChaikaS) [reply script.js]: done

**Pull request merged**